### DISCUSSION TITLE
Change the number formats for chart data points but not y-Axis

### DISCUSSION THREAD

**Discussion by mike23pizzo** [Kanaries/pygwalker#561] [Q&A] [answered]
I might just be missing this option, but prior to changing the number format for the chart the y-Axis is formatted with the thousands separator but the data points when hovering on the chart are not.  When I go to config > format, and enter "," into the Number format field, the numbers within the chart are formatted with the thousands separator now, but y-Axis changes to scientific notation.  I selected y-Axis under the "Independence" section (though I haven't been able to find documentation on that section) thinking maybe that will prevent config changes from applying to it, but it still changes to scientific notation.

Any help would be greatly appreciated, thanks in advance!

**Comment by longxiaofei** [comment]
It seems to be a bug caused by vega-lite, and the developers of graphic-walker are currently trying to solve it.

You can try to solve it temporary by forcing the specified valid decimal, example: `,.2f`.

**Reply by mike23pizzo** [reply, marked as answer]
Thank you for your reply.  I used `,.0f` and got my desired outcome, with the y-Axis not changing to scientific notation.

Should I mark this as "answered" or should it remained as "unanswered" since the bug has not been solved yet.  I'm new to all of this so I don't know what best practice is.
